docs/templates/index.md

```diff
@@ -0,0 +1,12 @@
+---
+---
+
+When writing docs, start with one of the five templates we provide:
```

Comment:
That'd be good for people wanting to contribute the docs. Probably under "Writing Guides" or "Writing Documents".